### PR TITLE
Add support for Lisk and Lisk Sepolia Testnet networks

### PR DESCRIPTION
**Pull request by sameersubudhi**
**Description**

This PR adds support for the `Lisk` and `Lisk Sepolia Testnet` networks so that tokens deployed on these networks can be automatically supported by [Optimism Bridges](https://app.optimism.io/bridge) when new tokens are submitted, verified and merged to the Superchain Token Registry.

**Tests**

- Not applicable; did not find any existing relevant tests

**Additional context**

- Official contracts list: https://docs.lisk.com/contracts

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -115,6 +116,8 @@ We currently accept tokens on the following chains:
 - `optimism-sepolia`
 - `mode`
 - `pgn`
+- `lisk`
+- `lisk-sepolia-testnet`
```
Comment (hamdiallam) [inline README.md:120]: can the "-testnet" suffix be removed? It's implicit with it being sepolia

File: src/chains.ts [hunk 2]
```diff
@@ -74,6 +85,14 @@ export const NETWORK_DATA: Record<Chain, Network> = {
     ),
     layer: 2,
   },
+  'lisk-sepolia-testnet': {
```
Comment (hamdiallam) [inline src/chains.ts:88]: same here as well

File: src/chains.ts [hunk 3]
```diff
@@ -110,16 +132,21 @@ export const L2_STANDARD_BRIDGE_INFORMATION: Record<
   'pgn-sepolia': {
     l2StandardBridgeAddress: '0x4200000000000000000000000000000000000010',
   },
+  'lisk-sepolia-testnet': {
```
Comment (hamdiallam) [inline src/chains.ts:135]: +1

File: src/chains.ts [hunk 4]
```diff
@@ -157,5 +188,9 @@ export const L1_STANDARD_BRIDGE_INFORMATION: Record<
       l2Chain: 'base-sepolia',
       l1StandardBridgeAddress: '0xfd0Bf71F60660E2f608ed56e1659C450eB113120',
     },
-  ]
+    {
+      l2Chain: 'lisk-sepolia-testnet',
```
Comment (hamdiallam) [inline src/chains.ts:192]: +1

File: src/schemas.ts [hunk 5]
```diff
@@ -62,11 +62,13 @@ export const TOKEN_DATA_SCHEMA = {
         ethereum: TOKEN_SCHEMA,
         optimism: TOKEN_SCHEMA,
         base: TOKEN_SCHEMA,
+        lisk: TOKEN_SCHEMA,
         mode: TOKEN_SCHEMA,
         pgn: TOKEN_SCHEMA,
         sepolia: TOKEN_SCHEMA,
         'base-sepolia': TOKEN_SCHEMA,
         'optimism-sepolia': TOKEN_SCHEMA,
+        'lisk-sepolia-testnet': TOKEN_SCHEMA,
```
Comment (hamdiallam) [inline src/schemas.ts:71]: here too

File: src/types.ts [hunk 6]
```diff
@@ -29,8 +29,20 @@ export type Chain =
   | 'pgn-sepolia'
   | 'sepolia'
   | 'mode'
+  | 'lisk'
+  | 'lisk-sepolia-testnet'
```
Comment (hamdiallam) [inline src/types.ts:33]: +1

File: src/types.ts [hunk 7]
```diff
@@ -29,8 +29,20 @@ export type Chain =
...
+const l2Chains = [
+  'optimism',
+  'optimism-sepolia',
+  'base',
+  'base-sepolia',
+  'pgn-sepolia',
+  'pgn',
+  'mode',
+  'lisk',
+  'lisk-sepolia-testnet',
```
Comment (hamdiallam) [inline src/types.ts:44]: +1

**Review by hamdiallam — COMMENTED** [review]
Looks great! Just 1 comment on the "-testnet" suffix

File: README.md (hunk 1 above)
Comment (sameersubudhi) [reply README.md]: Removed the `-testnet` suffix from all the places. Please re-review 🙏

**Review by hamdiallam — APPROVED** [review]
Thanks for updating!

**Pull request merged**